eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -247,49 +247,51 @@ jobs:
                      -configuration Debug | xcpretty -c
         displayName: 'Build examples [iphonesimulator]'
 
-#  - job: 'SmokeTest'
-#    strategy:
-#      matrix:
-#        # Build SDK libraries as frameworks, dynamically link to copies stored within the app bundle
-#        frameworks:
-#          PodsLinkage: frameworks
-#
-#        # Build SDK libraries as static libs (.a), statically link them into the app binary
-#        # This is currently broken, probably because Calling is pre-built as a framework
-#        # static:
-#        #   PodsLinkage: static
-#
-#        # Build SDK libraries as frameworks, but then statically link them into the app binary
-#        # Calling is still dynamically linked, probably because Calling is pre-built as a framework
-#        staticframeworks:
-#          PodsLinkage: staticframeworks
-#
-#    variables:
-#      - template: ../variables/globals.yml
-#
-#    pool:
-#      vmImage: '$(OSVmImage)'
-#
-#    steps:
-#      - script: sudo xcode-select --switch /Applications/Xcode_$(XcodeVersion).app
-#        displayName: 'Use Xcode $(XcodeVersion)'
-#
-#      - script: |
-#          cd examples/AzureSDKSmokeTest
-#          pod install
-#        env:
-#          PODS_LINKAGE: $(PodsLinkage)
-#        displayName: 'Install dependencies'
-#
-#      - script: |
-#          set -o pipefail
-#          cd examples/AzureSDKSmokeTest
-#          xcodebuild build \
-#                     -workspace AzureSDKSmokeTest.xcworkspace \
-#                     -scheme AzureSDKSmokeTest \
-#                     -sdk iphonesimulator \
-#                     -configuration Debug | xcpretty -c
-#        displayName: 'Build smoke test [iphonesimulator]'
+  - job: 'SmokeTest'
+    strategy:
+      matrix:
+        # Build SDK libraries as frameworks, dynamically link to copies stored within the app bundle
+        frameworks:
+          PodsLinkage: frameworks
+
+        # Build SDK libraries as static libs (.a), statically link them into the app binary
+        # This is currently broken, probably because Calling is pre-built as a framework
+        # static:
+        #   PodsLinkage: static
+
+        # Build SDK libraries as frameworks, but then statically link them into the app binary
+        # Calling is still dynamically linked, probably because Calling is pre-built as a framework
+        staticframeworks:
+          PodsLinkage: staticframeworks
+
+    variables:
+      - template: ../variables/globals.yml
+
+    pool:
+      vmImage: '$(OSVmImage)'
+
+    steps:
+      - script: sudo xcode-select --switch /Applications/Xcode_$(XcodeVersion).app
+        displayName: 'Use Xcode $(XcodeVersion)'
+
+      - script: |
+          cd examples/AzureSDKSmokeTest
+          pod install
+        env:
+          PODS_LINKAGE: $(PodsLinkage)
+          PODFILE_GIT_BRANCH: $(system.pullRequest.sourceBranch)
+          PODFILE_GIT_REPO: $(system.pullRequest.sourceRepositoryUri)
```

Comment:
@mitchdenny this is where I'm currently stuck. I also tried `Build.SourceBranch` and `Build.SourceBranchName` but those were even more off than what I currently have.